spring-cloud-gcp-data-spanner/src/test/java/org/springframework/data/spanner/core/mapping/BasicSpannerPersistentPropertyTests.java

```diff
@@ -38,15 +41,24 @@ public void testGetColumn() {
 				(BasicSpannerPersistentEntity<TestEntity>) (new SpannerMappingContext()
 				.getPersistentEntity(TestEntity.class));
 
-		Set<String> cols = new HashSet<>();
-		cols.add("id");
-		cols.add("custom_col");
-		cols.add("other");
+		MatcherAssert.assertThat(entity.columns(),
+				Matchers.containsInAnyOrder("id", "custom_col", "other"));
+	}
+
+	@Test(expected = MappingException.class)
+	public void testNullColumnName() {
+		SpannerMappingContext context = new SpannerMappingContext();
+		FieldNamingStrategy namingStrat = Mockito.mock(FieldNamingStrategy.class);
+		Mockito.when(namingStrat.getFieldName(ArgumentMatchers.any())).thenReturn(null);
+		context.setFieldNamingStrategy(namingStrat);
+		BasicSpannerPersistentEntity<TestEntity> entity =
+				(BasicSpannerPersistentEntity<TestEntity>) (context
+				.getPersistentEntity(TestEntity.class));
 
 		entity.columns().forEach(col -> {
 			BasicSpannerPersistentProperty prop = (BasicSpannerPersistentProperty) entity
 					.getPersistentPropertyByColumnName(col);
-			Assert.assertTrue(cols.contains(prop.getColumnName()));
+			prop.getColumnName();
```

Comment:
maybe?
`assertThat(prop.getColumnName(), isEmptyOrNullString());`

Reply:
That line `prop.getColumnName()` is the one that's supposed to throw the exception

Reply:
oh, my bad!

Reply:
maybe an inline comment here can clarify the intent.

Reply:
Changes made, PTAL